addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/handler/RFXComBridgeHandler.java

```diff
@@ -146,6 +148,13 @@ public void initialize() {
 
         configuration = getConfigAs(RFXComBridgeConfiguration.class);
 
+        if (configuration.serialPort != null && configuration.serialPort.startsWith("rfc2217")) {
+            logger.debug("Please use the Transceiver over TCP/IP bridge type for a serial over IP connection.");
```

Comment:
Is this limitation needed?

Reply:
Yes, it is needed if you don't want users loosing their time trying to use RFC2217 while it cannot work ;)
I already tried with the Powermax binding which is identical to the RFXCOM binding in the way to handle the serial connection and the binding was no more working. The reason is that our RFC2217 implementation does not provide a way to set a timeout on the read command. So the reading thread is blocked on a read() while at the same time another thread needs to write commands.
But you try yourself with RFC2217 if you think there is really a chance that it can work with a blocking read and the current binding architecture.
As the binding provides a TCP connector (of course with a non blocking read), there is no problem to disabling RFC2217 as serial over IP can simply be setup by using the TCP thing.